locale/de/LC_MESSAGES/django.po

```diff
@@ -0,0 +1,1250 @@
+msgstr ""
+
+#: app/models/project.py:26
+msgid ""
+"Whether this project has been marked for deletion. Projects that have "
+"running tasks need to wait for tasks to be properly cleaned up before they "
+"can be deleted."
+msgstr ""
+
+#: app/models/project.py:26
+#, fuzzy
+#| msgid "Delete"
+msgid "Deleting"
+msgstr "Löschen"
+
+#: app/models/project.py:56 app/models/task.py:220
+#, fuzzy
+#| msgid "First Project"
+msgid "Project"
+msgstr "Erstes Projekt"
+
+#: app/models/project.py:57
+#, fuzzy
+#| msgid "First Project"
+msgid "Projects"
+msgstr "Erstes Projekt"
+
+#: app/models/setting.py:21
+msgid "The name of your application"
+msgstr ""
+
+#: app/models/setting.py:21
+msgid "App name"
+msgstr ""
+
+#: app/models/setting.py:22
+msgid "A 512x512 logo of your application (.png or .jpeg)"
+msgstr ""
+
+#: app/models/setting.py:22
+msgid "App logo"
+msgstr ""
+
+#: app/models/setting.py:32
+msgid "The name of your organization"
+msgstr ""
+
+#: app/models/setting.py:32
+msgid "Organization name"
+msgstr ""
+
+#: app/models/setting.py:33
+msgid "The website URL of your organization"
+msgstr ""
+
+#: app/models/setting.py:33
+msgid "Organization website"
+msgstr ""
+
+#: app/models/setting.py:34 app/models/theme.py:48 app/models/theme.py:49
+#: app/templates/app/logged_in_base.html:95
+msgid "Theme"
+msgstr "Thema"
+
+#: app/models/setting.py:35
+msgid "Active theme"
+msgstr ""
+
+#: app/models/setting.py:74 app/templates/app/logged_in_base.html:101
+msgid "Application"
+msgstr "Anwendung"
+
+#: app/models/setting.py:77 app/models/setting.py:78
+msgid "Settings"
+msgstr ""
+
+#: app/models/task.py:217
+msgid "Id"
+msgstr ""
+
+#: app/models/task.py:219
+msgid "Identifier of the task (as returned by NodeODM API)"
+msgstr ""
+
+#: app/models/task.py:219
+msgid "UUID"
+msgstr ""
+
+#: app/models/task.py:220
+msgid "Project that this task belongs to"
+msgstr ""
+
+#: app/models/task.py:221
+msgid "A label for the task"
+msgstr ""
+
+#: app/models/task.py:222
+msgid ""
+"Number of milliseconds that elapsed since the beginning of this task (-1 "
+"indicates that no information is available)"
+msgstr ""
+
+#: app/models/task.py:222
+msgid "Processing Time"
+msgstr ""
+
+#: app/models/task.py:223
+msgid ""
+"Processing node assigned to this task (or null if this task has not been "
+"associated yet)"
+msgstr ""
+
+#: app/models/task.py:223
+msgid "Processing Node"
+msgstr ""
+
+#: app/models/task.py:224
+msgid ""
+"A flag indicating whether this task should be automatically assigned a "
+"processing node"
+msgstr ""
+
+#: app/models/task.py:224
+msgid "Auto Processing Node"
+msgstr ""
+
+#: app/models/task.py:225
+msgid "Current status of the task"
+msgstr ""
+
+#: app/models/task.py:225
+msgid "Status"
+msgstr ""
+
+#: app/models/task.py:226
+msgid "The last processing error received"
+msgstr ""
+
+#: app/models/task.py:226
+msgid "Last Error"
+msgstr ""
+
+#: app/models/task.py:227
+msgid "Options that are being used to process this task"
```

Comment:
"that are being" seems superfluous.

Reply:
True! Open a PR that changes task.py:227? :pray:

Reply:
https://github.com/OpenDroneMap/WebODM/pull/944